package.json

```
@@ -55,7 +55,7 @@
     "typescript": ">=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta"
   },
   "dependencies": {
-    "tslib": "^1.8.1"
+    "tslib": "^1.8.1 || ^2.0.0"
```

I think this should just be "^2.0.0" since the package is being built with typescript >=3.92
```suggestion
    "tslib": "^2.0.0"
```

---

The dependency includes more wide range of tslib so I think we have to includes the old ones?

---

tslib is a dependency because when this project gets built, the resulting javascript will contain imports to helper functions from the tslib module. As a result, the version of typescript used to build the package needs to match with the corresponding version of tslib that it was expecting. Going back to the original issue, its injecting code that expects `__spreadArray` to be defined in tslib, but that helper function doesn't exist in earlier versions of tslib.

I guess the way to think about it is that the code generated by the typescript compiler has a dependency on a particular version of the tslib library, but the typescript compiler itself doesn't have any hard dependencies on tslib itself to be able to use its api.